nbgrader/apps/formgradeapp.py

```diff
@@ -54,8 +47,11 @@ class FormgradeApp(BaseNbGraderApp):
     ip = Unicode("localhost", config=True, help="IP address for the server")
     port = Integer(5000, config=True, help="Port for the server")
     base_directory = Unicode('.', config=True, help="Root server directory")
-    directory_format = Unicode('{notebook_id}.ipynb', config=True, help="Format string for the directory structure of the autograded notebooks")
-    start_nbserver = Bool(True, config=True, help="Start a single notebook server that allows submissions to be viewed.")
+    directory_format = Unicode('{notebook_id}.ipynb', config=True, help="""Format
+        string for the directory structure of the autograded notebooks""")
+    authenticator_class = DottedObjectName('nbgrader.auth.noauth.NoAuth', config=True, help="""
```

Comment:
you can use `authenticator_class = Type('nbgrader.auth...')` and avoid needing the DottedObjectName + import_string.